jenkins/release-manifest-commit-lock/release-manifest-commit-lock.jenkinsfile

```diff
@@ -47,7 +47,7 @@ pipeline {
         )
         string(
             name: 'COMPONENTS',
-            description: 'The selected components from the manifest to update the commit ID',
+            description: 'The selected components from the manifest to update the commit ID. Use comma seperate values, example OpenSearch,ml-commons,cross-cluster-replication',
```

Comment:
Add a period at the end of the sentence.

Reply:
Sure dB will do.

Reply:
Related PR https://github.com/opensearch-project/opensearch-build/pull/4680.